### ISSUE TITLE
Responsive issue on homepage 

### ISSUE THREAD

**Issue by vishalvivekm** [layer5io/layer5#6069]
### Description

on homepage https://layer5.io
![Screenshot_20241108-173614](https://github.com/user-attachments/assets/c82d2404-6479-4e65-ab3a-99285850d395)

### Environment:
- Host OS:
- Browser:

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by vishalvivekm** [comment]
On kanavs page https://layer5.io/kanavs, the logo on top right is is out of view.
![Screenshot_20241108-174353](https://github.com/user-attachments/assets/40dc1715-eaed-490f-892c-5bfb9d644ef8)

**Comment by NishantSinghhhhh** [comment]
@vishalvivekm  , Sir in the first issue , I have to centrally align the the division of moving icons , as few of the divisons are aligned towards to the left side

**Comment by vishalvivekm** [comment]
> @vishalvivekm , Sir in the first issue , I have to centrally align the the division of moving icons , as few of the divisons are aligned towards to the left side

yeah, that would be great. I was pointing out the text being cutoff from top, please open the page once in mobile device @NishantSinghhhhh and you'll find it.

**Comment by NishantSinghhhhh** [comment]
Okay sure Sir

**Comment by upsaurav12** [comment]
HI @vishalvivekm i have seen responsive issue on https://docs.layer5.io/ at ``<=951px``
https://github.com/user-attachments/assets/0656128e-6b66-4092-9507-db30aa035409

**Comment by radhathakare** [comment]
Hi @vishalvivekm @sudhanshutech, can I work on this task if it's still open?

**Comment by Souradip121** [comment]
![Image](https://github.com/user-attachments/assets/ebfad031-105e-40fb-9b21-bde3f04d6612)
The  https://docs.layer5.io/ page is responsive; it would be best to close the issue right now!
